Riot/Modules/Spaces/SpaceList/SpaceListViewCell.swift

```diff
@@ -61,10 +61,12 @@ final class SpaceListViewCell: UITableViewCell, Themable, NibReusable {
         if viewData.isInvite {
             self.badgeLabel.isHidden = false
             self.badgeLabel.badgeColor = ThemeService.shared().theme.colors.alert
```

Comment:
Didn't notice that we use the shared instance here. We usually use a `private var theme: Theme?` in views for this case. Value updated in `update(theme: Theme)`